calicoctl/commands/node/diags.go

```diff
@@ -155,7 +159,39 @@ such as transfer.sh using curl or similar.  For example:
 
 }
 
-// writeDiags executes the dignostic commans and outputs the result in the file
+// getDockerLogs will attempt to grab logs for any "calico" named containers for hosted installs.
+func getDockerLogs(logDir string) {
+	os.Mkdir(logDir, os.ModeDir)
+
+	fmt.Println("Copying logs from Calico containers")
+	// Get a list of Calico containers running on this Node.
+	out, err := exec.Command("docker", "ps", "-a", "--filter", "\"name=calico\"", "--format", "{{.Names}}").Output()
+	if err != nil {
+		fmt.Printf("Could not run docker command: %s\n", err)
```

Comment:
does it make sense to capture these failures (170 and 175) in the diag output? it's possible the errors are too unlikely to bother trying to capture.